README.md

```diff
@@ -1,70 +1,70 @@
-# Getting Started with Create React App
+# The GrapeVine 🍇
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Jacqueline Cope & Nga Dang
 
-## Available Scripts
+🌟 Deployed App: https://thegrapevine.netlify.app/
 
-In the project directory, you can run:
+🖥 Client Repo: https://github.com/TheGrapeVineCo/client
 
-### `npm start`
+🖥 Server Repo: https://github.com/TheGrapeVineCo/Server
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+📖 Documentation Repo: https://github.com/TheGrapeVineCo/The-GrapeVine-Docs
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+The GrapeVine is a site that aims to bring people together on a dedicated platform where they can share their love of Australian wine and learn a tip or two along the way. This benefits the community of wine consumers at all levels to then be able to try recommended wines by those who have shared and contributed within the community. The application has been built with React and Rails and deployed with Netlify and Heroku.
 
-### `npm test`
+## Installation Instructions
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+To use the production app, please visit https://thegrapevine.netlify.app/ and signup to get started. Alternatively, to use the app locally, please follow the below instructions.
 
-### `npm run build`
+**Requirements:**
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+Rails 6.1.6
+Ruby 3.0.2
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+After cloning the repository locally and moving into `The GrapeVine` root directory:
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+_Server_
 
-### `npm run eject`
+- Install required dependencies with `$ bundle install`
```

Comment:
Should word 'dependencies' be replaced by 'gems' ? This isn't show stopper though but just a suggestion.